mockup1/assets/filtering.js

```diff
@@ -62,13 +62,10 @@
 
     const total = document.querySelectorAll(SELECTOR_PREFIX);
     const specs = document.querySelectorAll(SELECTOR_PREFIX + '[style*="opacity: 1"]');
-    if (selectors.length > 0)
-      summary.innerHTML = specs.length + ' spec' + (1 === specs.length ? '' : 's') + ' (of ' + (total.length) + ')';
-    else
-      summary.innerHTML = total.length + ' specs (no filters)';
+    summary.innerHTML = specs.length + ' specifications' + (1 === specs.length ? '' : 's') + ' (total of ' + total.length + ' specifications, including retired documents)';
```

Comment:
extra 's'
also line 66 of the index.html file you  got "specs".
For consistency I think we should use "specifications" too.

Reply:
addressed in https://github.com/w3c/tr-pages/pull/83/commits/ec5600fed4dc767beaf114dc535953ea8aa6db1d